components/tabs/src/sync/record.rs

```diff
@@ -400,4 +400,83 @@ pub(crate) mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_tabs_record_from_payload_ttl_values() -> Result<()> {
+        struct TestCase {
+            case_description: String,
+            data: JsonValue,
+        }
+
+        let test_cases = vec![
+            TestCase {
+                case_description: "test invalid ttl".to_string(),
```

Comment:
do we want to be specific here about the test case, like "test no ttl provided" etc? Same goes for the other test cases

Reply:
This is a copy/paste omission. I'll make these more specific--thanks.